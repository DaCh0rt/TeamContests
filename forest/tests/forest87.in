301 246
-755 9195
4714 6756
9355 4462
8229 7793
3157 -6090
2635 -3756
-9396 2310
-7406 -2794
-6175 301
-1269 5675
-1482 -1020
4116 -5979
-8516 9790
6536 -6008
-105 2419
2834 -1514
6566 1495
-5638 4812
6651 8499
-9899 7252
884 -5327
5998 -6230
-3813 721
-4017 7964
-2360 -4196
7237 2155
685 9318
-7193 4908
2034 -3776
5342 -1143
402 -4643
-8265 -4855
1863 -4309
9424 -6951
9957 4247
2232 -1845
-3218 -9079
-4884 -9799
9520 5969
-9631 -422
7673 6845
-7888 -100
9835 -5102
-9895 -4116
-5308 3642
8289 -9332
1381 -6727
-9643 -7120
-3904 -6912
-8466 -3183
-6551 1046
-7420 -1127
633 9851
-4235 7655
2779 -3472
-6722 -8204
-2455 6
5237 5741
-3813 -2367
2479 8132
-5409 -5996
6773 1911
4358 -3804
-2886 4789
-3510 -6276
-9092 3197
-9925 -8665
8891 9486
-816 7626
-4342 131
-3631 -7699
5366 -723
7089 -7168
-6901 38
-4130 -2883
2680 5591
140 2420
6212 8004
-1125 3744
8122 -4307
-9024 -870
638 -2747
2700 5280
7795 -605
-7877 6709
1595 4008
-1530 -3544
3908 8403
8579 7498
1414 -7745
-103 -1087
-4966 316
8227 -1724
6028 -887
7291 9107
3859 3871
-289 4871
8438 1699
8571 -9161
-603 3690
-5429 1972
5251 -1652
7761 1430
-5394 2591
9039 363
-162 2611
-6818 -8057
1980 2361
-7279 -4373
3937 -3580
-365 -8491
-8196 6617
3058 6446
-9313 9285
8858 -2182
5964 -6407
3763 4553
2680 8470
-149 9748
-7472 -4460
7094 -7541
3018 2002
-3139 -2770
6796 -3565
-1545 -8342
-9429 -3146
-8879 9400
2399 -6480
4463 9356
-6006 -5705
-1531 -4477
-3644 -9826
6801 4920
-299 239
9951 436
-4796 -8133
-7384 -5322
3040 -5542
3288 8144
-7847 4433
-3675 -6347
-2733 8343
-5854 -8164
-9315 -6753
-618 5223
-3902 4148
-4042 -2418
-2111 -4565
-2899 -8319
1973 -7281
-2730 -4382
-4666 1106
4988 2860
-8741 6013
-9689 -3641
5419 -6873
-3000 2877
7578 4834
4899 -7351
-7574 -3783
-9341 33
-186 -7245
2706 -3852
-9297 -3903
7515 2423
5139 2126
-9303 -5516
9750 -8725
-8524 -8648
-6422 -8954
5020 2544
8277 -6198
1233 -7605
2373 -4994
-4407 -7273
-2616 770
-1271 3169
3656 6754
4208 3750
-316 1298
460 -8013
9095 900
-3857 -7356
-7831 3345
-6813 -5408
9340 2009
6881 3070
7877 -3186
-6965 -6346
-7368 -5516
3206 -7502
8893 -391
-4237 -4482
5857 1548
7291 -1479
-1369 -5411
-7283 5599
6765 -1120
180 -7680
-3885 -7103
4105 -5977
4158 -7339
-7019 -419
-1199 -7796
-7723 -849
9148 2784
-2948 -6966
3040 -8742
6836 7551
3929 -608
-4661 6810
3683 -27
2870 -6886
-9026 5312
-8388 -8652
-2859 448
7802 6187
-2193 -9860
9413 9376
-1389 -4785
-3485 5813
-4451 9011
9110 -1593
7910 3811
2614 -628
316 -3723
8857 2668
-4055 -7322
4755 8601
-4536 -2594
3494 -1254
-2114 -3401
-569 9295
5928 -4245
7651 9678
-9837 2181
-9017 4323
1171 -6115
315 -1293
7604 6630
-2326 -6959
6115 -349
2088 9628
-5046 -5418
2833 9775
683 9246
5904 -5027
-5932 -3063
-2870 1160
4508 -6845
9695 9823
9216 -3054
-4354 -9157
1330 -9747
2081 -507
-9010 7821
-2233 5255
2269 4461
1248 7078
8915 9745
-4221 5826
3445 -2634
1261 5546
5169 8701
7692 -988
-5635 -2914
-7974 136
-7994 4459
2906 4563
-2776 2765
8412 1501
3530 -7130
7964 -4554
5037 5444
-6871 195
4478 -8756
7705 8622
2986 364
9558 120
7635 -6697
-8692 -6265
-8805 -1327
-5147 -9470
-1730 8415
9807 2938
1003 -893
2614 -9223
6271 -6484
6340 7420
-6777 -7316
1374 9808
-791 -4002
-1332 8138
-813 64
-7106 -6918
720 781
-582 -5493
3224 -9959
650 -1226
9712 7027
889 -4503
