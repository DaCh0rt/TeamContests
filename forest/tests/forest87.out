77251.90 401634809.90
